<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>711</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(249, 247, 247);
color: rgb(63, 114, 175);
border-radius: 7px;
border: 0.7px solid black;</string>
    </property>
    <property name="title">
     <string>Optimize Noun Application:</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 0px;
border: 0px;</string>
     </property>
     <property name="text">
      <string>Input string:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditInput">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>551</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(219, 226, 239);
color: rgb(17, 45, 78);
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditOutput">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>551</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(219, 226, 239);
color: rgb(17, 45, 78);
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 0px;
border: 0px;</string>
     </property>
     <property name="text">
      <string>Result:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOptimize">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(219, 226, 239);
color: rgb(17, 45, 78);
border-radius: 15px;</string>
     </property>
     <property name="text">
      <string>Optimize</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/ic_convert.png</normaloff>../images/ic_convert.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonNew">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>160</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(219, 226, 239);
color: rgb(17, 45, 78);
border-radius: 15px;</string>
     </property>
     <property name="text">
      <string>New</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/ic_new.png</normaloff>../images/ic_new.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonExit">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>160</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(219, 226, 239);
color: rgb(17, 45, 78);
border-radius: 15px;</string>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/ic_exit.png</normaloff>../images/ic_exit.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>786</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
